### PR TITLE
EPMLSTRCMW-253 Feat: Implement GET endpoint to fetch project file by project id and projectFile name

### PR DESCRIPTION
**Pull request by DruzhininPavel**
DESCRIPTION
implemented a GET endpoint to get a project file by project id and project file name

** CHANGES **
Added GET endpoint to ProjectFileController.scala
Wrote three tests in ProjectFileControllerTest.scala for the test get file

### CODE REVIEW

**Review by OlegKoshelev — APPROVED** [review]
Well done!

**Review by Nikolai-Isaev — APPROVED** [review]
Lgtm

**Review by gominasian — APPROVED** [review]

File: controllers/src/test/scala/cromwell/pipeline/controller/ProjectFileControllerTest.scala [hunk 1]
```diff
@@ -81,53 +84,83 @@ class ProjectFileControllerTest extends AsyncWordSpec with Matchers with Scalate
...
         }
       }
     }
+
+    "get file" should {
+      "return file with status code OK" in {
+        when(projectFileService.getFile(projectId, path, versionOption, accessToken.userId))
+          .thenReturn(Future.successful(projectFile))
+        Get(s"/files?project_id=${projectId.value}&project_file_path=$pathString&version=$versionString") ~> projectFileController
+          .route(accessToken) ~> check {
```
Comment (myazinn) [inline controllers/src/test/scala/cromwell/pipeline/controller/ProjectFileControllerTest.scala]:
```suggestion
        Get(s"/files?project_id=${projectId.value}&project_file_path=$pathString&version=$versionString") ~>
          projectFileController.route(accessToken) ~> check {
```

File: services/src/main/scala/cromwell/pipeline/service/ProjectFileService.scala [hunk 2]
```diff
@@ -40,6 +40,19 @@ class ProjectFileService(
...
+    path: Path,
+    version: Option[PipelineVersion] = None,
+    userId: UserId
+  ): Future[ProjectFile] =
+    projectService.getUserProjectById(projectId, userId).flatMap { project =>
+      projectVersioning.getFile(project, path, version).flatMap {
+        case Left(versioningException) => Future.failed(versioningException)
+        case Right(projectFile)        => Future.successful(projectFile)
+      }
+    }
```
Comment (myazinn) [inline services/src/main/scala/cromwell/pipeline/service/ProjectFileService.scala:54]: We should add tests for this method too

**Review by myazinn — CHANGES_REQUESTED** [review]
Nice work!
Please add a bit more tests, other than that everything looks cool

File: controllers/src/test/scala/cromwell/pipeline/controller/ProjectFileControllerTest.scala (hunk 1 above)
Comment (DruzhininPavel) [reply controllers/src/test/scala/cromwell/pipeline/controller/ProjectFileControllerTest.scala]: Thaks, I`ll fix it.

File: services/src/main/scala/cromwell/pipeline/service/ProjectFileService.scala (hunk 2 above)
Comment (DruzhininPavel) [reply services/src/main/scala/cromwell/pipeline/service/ProjectFileService.scala:54]: Yes, I really forgot to do it. Thank you for reminding.!

File: controllers/src/test/scala/cromwell/pipeline/controller/ProjectFileControllerTest.scala (hunk 1 above)
Comment (DruzhininPavel) [reply controllers/src/test/scala/cromwell/pipeline/controller/ProjectFileControllerTest.scala]: Fixed.

File: services/src/main/scala/cromwell/pipeline/service/ProjectFileService.scala (hunk 2 above)
Comment (DruzhininPavel) [reply services/src/main/scala/cromwell/pipeline/service/ProjectFileService.scala:54]: I wrote 3 additional tests

**Review by myazinn — APPROVED** [review]

**Review by Dmitrii-Shevchenko — APPROVED** [review]

**Pull request merged**